### DISCUSSION TITLE
Digital Flow meter

### DISCUSSION THREAD

**Discussion by NWSAlpine** [tagyoureit/nodejs-poolController#928] [Nixie]
What if any options can we use to read real time GPM with a full nixie controller setup?

**Comment by tagyoureit** [comment]
There are very few that I know of.  You would need a 3" (or 4" depending on your piping) flow meter that has the appropriate outputs.  I remember a few people investigating this years ago and the flow meters were prohibitively expense.  

Understanding flow seems like it would be a great way to manage your pool, but at the end of the day it doesn't tell you much.  The best approach is to run the pumps at the slowest speeds possible for the longest period of time because that keeps costs way down.  If the pool is struggling to maintain proper filtration after a few days, but up the RPM's slightly and watch it again.

**Reply by NWSAlpine** [reply]
I agree that it's not really necessary but thought I would ask. The only digital option I saw was the Flowviz which was $500+ for the controller only. 

I just want to get a baseline for optimal RPM/GPM for my heater and different modes. That being said it might not even be worth it for that. I chlorinate fine on 1000 rpm with my Pentair VSF pulling 55 watts.

**Comment by gw8674** [comment]
I have a FlowVis Digital installed in the return side of the plumbing and I am feeding the 4-20mA values from the FlowVis into REM via my MEGA-IND. I have the 4-20mA conversion factors for it as well. Unfortunately at this time, there is no transport mechanism established to feed the converted values (4-20 value to GPM), back to njspC/dashPanel. 

Same with my PD6300 Pool Auto Refill Flow Meter/Totalizer.
<img width="609" alt="2024-04-23_21-32-11" src="https://github.com/tagyoureit/nodejs-poolController/assets/52094163/afb93e8c-254d-48f9-9e3c-746becc2e9d3">

I think this was on our "wish list" at one time but probably low on the priority list.

**Reply by jeffegg** [reply]
I have a local change to REM for a generic device for flow meters to do the GPM conversions from mA as I have the flowviz as well. I have not had the time to get the njsPC side done. I could put a pull request in for this change to add the generic device if there is interest.
![Screenshot_20240423-213847.png](https://github.com/tagyoureit/nodejs-poolController/assets/1361872/33fc5e49-c6cc-45e0-83eb-06b884de87f1)

**Reply by NWSAlpine** [reply]
What parts would we need to replicate this setup? Do you still need to flowviz controller or just the flow meter like the FV-C wired directly to the sequent systems card?

**Comment by gw8674** [comment]
@jeffegg 
Yea, I see what you did there. That looks great! All I need is the ability to do that in REM for both my FlowVis and PD6300 Flow meter and then get those values over to njspC and I'd be set. That would be awesome with whatever you can do!
Thanks much!!
